src/main/java/com/laytonsmith/PureUtilities/Point3D.java

```diff
@@ -9,7 +9,7 @@
 	/**
 	 * A Point3D located at [0, 0, 0] (zero)
 	 */
-	public static final Point3D ZERO = new Point3D();
+	public static final Point3D ZERO = new Point3D(0, 0, 0);
```

Comment:
Ah, we had a misunderstanding then. I didn't quite understand why I should not remove it. I'll remove them then.